### ISSUE TITLE
华为安全自助平台扫描出https安全问题

### ISSUE THREAD

**Issue by yaok2012** [google/volley#255]
证书验证方法可能存在安全风险。危险等级：低		
Threat Level	Problem File	Identity Info
Low	com/android/volley/toolbox/HurlStack.java	HurlStack.openConnection(URL, Request) : HttpURLConnection;

请问会有计划去修改吗

**Comment by jpd236** [comment]
(I've used Google Translate to translate your bug, so apologies if I've misinterpreted anything)

I'm not familiar with this Huawei software, but there's no inherent security bug that I'm aware of here. If you have more details on what these security risks actually are, we can take a look.

HttpURLConnection does use the platform's implementation, which means it could potentially have security bugs, particularly on older versions of Android. If that's a concern for you, don't use HurlStack - instead use an HttpStack which uses an HTTP library that you bundle with your application and can update along with your application in the event of any security bugs or other updates.
